osgeo_importer/models.py

```diff
@@ -163,7 +163,7 @@ def file_url(self):
         """
         Exposes the file url.
         """
-        return '' # self.uploadfile_set.first().file.url
```

Comment:
not important, but some of these one-off flake8 fixes can be squashed with the changes made, or squashed with each other.